src/core/renderController.ts

```diff
@@ -18,10 +18,10 @@ module Plottable {
    * ```
    */
   export module RenderController {
-    var _componentsNeedingRender = new Utils.Set<Component>();
-    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
-    var _animationRequested = false;
-    var _isCurrentlyFlushing = false;
+    var componentsNeedingRender = new Utils.Set<Component>();
+    var componentsNeedingComputeLayout = new Utils.Set<Component>();
+    var animationRequested = false;
+    var isCurrentlyFlushing = false;
```

Comment:
Having the underscore convention here is similar to having underscores for our local variables within function definitions.  The intention of the underscore was so if a user were to call like object._someFunction, they would be signaled that the function usage as at your own risk.  However, since these variables are not even exported, there is no RenderController._componentsNeedingRender being shown to the user.